debian/stocks-server.postrm

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -e
+set -x
+
+case "$1" in
+	purge)
+		rm -rf /usr/share/stocks-server/root/CA
```

Comment:
I guess purging is _really really_ removing everything. Without CA users can no longer log in.

Reply:
The definition of purge:

```
       purge
           purge is identical to remove except that packages are removed and purged (any
           configuration files are deleted too).
```

It isn't clear what happens to state (we could check what `postgres-server` does in that case). I chose this interpretation for no particular reason.